internal/bridge/quantity.go

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2024 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package bridge
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func FromCPU(n int64) *resource.Quantity {
```

Comment:
We're not using this right now, are we?

Reply:
No, we're not. It would be useful if we ever decide to include CPU on the status... Not sure if I should keep it or just delete it for now.

Reply:
Yeah, I'm on the fence...